libs/scripts/scripts.js

```diff
@@ -53,6 +53,8 @@ async function loadPage() {
   await loadLazy([...navs, ...blocks]);
   const { default: loadModals } = await import('../blocks/modals/modals.js');
   loadModals();
+  const { getIconLibrary } = await import('../utils/decorate.js');
```

Comment:
Will every page have icons?  If not, we should wait to load until we know we need the iconLib.

Reply:
I updated this to conditionally load icons request based on block type. Let me know what you think. It sounds like the token source discussion is happening and this will likely move the association here to the token.icons sheet. obj

Reply:
👍 Now that we're going to have all the tokens in one file, it's probably fine to always load it.

Reply:
The usage on these is very minimal. I think we should keep it only conditionally loaded based on 'ICON_BLOCKS' types for now. Good topic to discuss in future global icon refactor task. let me know though if anything looks to need changed.